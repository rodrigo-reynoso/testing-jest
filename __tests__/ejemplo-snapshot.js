const cliente ={
    nombre:'Ignacio modificado 2',
    tipo:'premium',
    balance:500
};

describe('Testing al objeto con snapshot, y actualizando cuando se modifica el objeto',()=>{
    test('El cliente es Ignacio',()=>{
        expect(cliente).toMatchSnapshot();
    });
});