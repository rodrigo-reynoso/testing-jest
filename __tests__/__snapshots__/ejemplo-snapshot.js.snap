// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al objeto con snapshot, y actualizando cuando se modifica el objeto El cliente es Ignacio 1`] = `
Object {
  "balance": 500,
  "nombre": "Ignacio modificado 2",
  "tipo": "premium",
}
`;
